docs/design/coreclr/jit/ryujit-overview.md

```diff
@@ -759,7 +759,7 @@ phase-specific information by searching for the phase name. Some useful points f
 You can enable dumps by setting the `COMPlus_JitDump` environment variable to a space-separated list of the method(s)
 you want to dump. For example:
 
-```cmd
+```bat
```

Comment:
FYI both `cmd` and `bat` are correct. this change is not needed.

```cmd
:: with ```cmd

setx FOO=BAR
echo %FOO%
```

Reply:
fine I'll make them cmd consistently.